R/summarizeResults.R

```diff
@@ -0,0 +1,113 @@
+
+#' Internal function to summarize the results of the DQD run.
+#' 
+#' @param connectionDetails         A connectionDetails object for connecting to the CDM database
+#' @param cdmDatabaseSchema         The fully qualified database name of the CDM schema
+#' @param checkResults              A dataframe containing the results of the checks after running against the database
+#' @param cdmSourceName             The name of the CDM data source
+#' @param outputFolder              The folder to output logs and SQL files to
+#' @param outputFile                (OPTIONAL) File to write results JSON object 
+#' @param startTime                 The system time the check was started
+#' @param tablechecks               A dataframe containing the table checks
+#' @param fieldChecks               A dataframe containing the field checks
+#' @param conceptChecks             A dataframe containing the concept checks
+#' @param metadata                  Information from the CDM_SOURCE table with details about the database
+#' 
+#' @keywords internal
+#' 
+
+.summarizeResults <- function(connectionDetails,
+                              cdmDatabaseSchema,
+                              checkResults,
+                              cdmSourceName,
```

Comment:
Arguments `connectionDetails`, `cdmDatabaseSchema` and `cdmSourceName` are not used in this function.

Reply:
Before I remove these in the PR we should test it. In one of the other functions it looked like some parameters were not used. When I removed them the whole thing broke. I think it has to do with the cluster apply and lapply usage throughout the package. They do some things under the covers that is not made explicit in the code.